src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -57,8 +57,8 @@ trait FireCloudApiService extends CookieAuthedApiService
   with WorkspaceApiService
   with NotificationsApiService
   with MethodConfigurationApiService
-  with BillingService
-  with SubmissionService
+  with BillingApiService
```

Comment:
The standard for API route handlers seems to be:
* named ...ApiService
* lives in the webservice package

So I changed these 3 to fit